ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js

```diff
@@ -126,7 +127,11 @@ const SnapUIRendererComponent = ({
       initialState={initialState}
       context={context}
     >
-      <Box className="snap-ui-renderer__content" height={BlockSize.Full}>
+      <Box
+        className="snap-ui-renderer__content"
+        height={BlockSize.Full}
+        backgroundColor={BackgroundColor.backgroundAlternative}
```

Comment:
We need to solve this before merging.

<img width="375" alt="Screenshot 2024-08-28 at 12 00 09" src="https://github.com/user-attachments/assets/73f5ea8d-79f0-4dde-bd20-a4b684b75f74">

Reply:
This should be fixed now by the latest changes I pushed.
![Screenshot 2024-08-29 at 12 31 52](https://github.com/user-attachments/assets/95a34e51-0c5d-4650-9468-49f5fdc7e94d)